apps/patient/src/views/Status.tsx

```diff
@@ -15,6 +26,67 @@ import { formatAddress, getFulfillmentType, preparePharmacy } from '../utils/gen
 import { orderStateMapping as m, text as t } from '../utils/text';
 import { useOrderContext } from './Main';
 import { HorizontalStatusStepper } from '../components/HorizontalStatusStepper';
+import dayjs from 'dayjs';
+
+function formatReadyTime(baseTime: string, minutesUntilReady: number): ReactElement {
+  const currentTime = dayjs(baseTime);
```

Comment:
This naming is confusing to me also. I keep reading `currentTime` as "now" when its really the `baseTime` just as a `dayjs`.

Reply:
yeah this went thru a few revisions, goods points, I'll address